.gitlab-ci.yml

```
@@ -70,6 +72,8 @@ trigger:mender-dist-packages:
   stage: trigger
   rules:
     - if: '$CI_COMMIT_BRANCH == "master"'
+    - if: $CI_PIPELINE_SOURCE == "schedule"
```

```suggestion
    - if: '$CI_PIPELINE_SOURCE == "schedule"'
```
please align these quotes to be consistent with the rest of the file

---

Right, thanks for caring about these details :)